src/transaction.js

```diff
@@ -1,14 +1,11 @@
 import map from 'lodash/map';
-import each from 'lodash/each';
-import isString from 'lodash/isString';
 import xdr from './generated/stellar-xdr_generated';
 import { hash } from './hashing';
 
 import { StrKey } from './strkey';
 import { Operation } from './operation';
-import { Network } from './network';
 import { Memo } from './memo';
-import { Keypair } from './keypair';
+import { TransactionBase } from './transaction_base';
 
 /**
  * Use {@link TransactionBuilder} to build a transaction object, unless you have
```

Comment:
I think it is worth adding a comment that we generally discourage creating `Transaction` instances manually. Instead people should be using `builder.fromXdr()` to construct `Transaction` instances. The same comment applies to `FeeBumpTransaction` as well.

If this were implemented in typescript I think we'd make `Transaction` and `FeeBumpTransaction` would be interfaces which are exported but the implementations would not be exported and only be local to the builder code

Reply:
👍